### DISCUSSION TITLE
Save the original utf-8 text and not in unicode format, inside JsonProperty

### DISCUSSION THREAD

**Discussion by alexiosgkikas** [neo4j-contrib/neomodel#748] [Q&A]
##  Description
Hello,
I am using the neomodel library to save some nodes in a Neo4j database. I have defined a class (type of NodeStructure) with one JSONProperty.
When I try to save a new node or update it, if this property contains utf-8 characters(i.e "τεστ"), it will be saved with unicode format (i.e. "\u03b1\u03bb\u03b5\u03be"). 
Upon retrieval of the nodes(with neomodel library), the text is back to normal. 
But, when I open the Neo4j Desktop to navigate on the database, the text is shown in unicode format. I would like to see the original text, which is more user-friendly.
By altering the JSONProperty function deflate, as shown below, I might be able to get the desired functionality.

```
   @validator
    def deflate(self, value):
        return json.dumps(value,ensure_ascii=False)

```
Note, that the text will be preserved exactly as it is if I use the StringProperty.

My questions:

1. Is there any other way to save the original text without modifying the library?
2. Could, the above modification, cause any issues in the functionality of the library?
3. Does the default behavior work as expected or is it a bug(since StringProperty doesn't convert the text)?

## How to Reproduce

1. Create the below model
```
   class TestJson(StructuredNode):
     json_field = JSONProperty()
```
5. Create a Node
```
t = TestJson(json_field = {"test":"τεστ"})
t.save()
```
6. Open the Neo4j Browser and check the Node's values

**Comment by aanastasiou** [comment]
Thank you for bringing this up, happy to review and consider a PR on this change.

The only hesitation I would have against `ensure_ascii=False` would be that the encoding would now be up to the system settings and I am not sure how much this would guarantee that data would be readable across systems and data entry methods.
